CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 # 1.0.0
 
 - Initial version of `safe-publish` script.
+
+# 1.1.0
```

Comment:
We usually put the latest version at the top of the file, could you please move this to the top?

Reply:
Sure will do.